db/schema.rb

```diff
@@ -38,6 +38,33 @@
     t.index ["weather_id"], name: "index_hourlies_on_weather_id"
   end
 
+  create_table "location_trails", force: :cascade do |t|
+    t.bigint "location_id"
+    t.bigint "trail_id"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.index ["location_id"], name: "index_location_trails_on_location_id"
+    t.index ["trail_id"], name: "index_location_trails_on_trail_id"
+  end
+
+  create_table "locations", force: :cascade do |t|
+    t.string "name"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.string "forecast_summary"
+    t.string "forecast_temp"
```

Comment:
Because I already had a model object called `forecast` that was being used in future iterations of the project I decided to assign temp data to a location